Gifski/GifskiWrapper.swift

```diff
@@ -58,10 +58,6 @@ final class GifskiWrapper {
 		self.pointer = pointer
 	}
 
-	deinit {
-		gifski_drop(pointer)
```

Comment:
@kornelski This was removed, so I guess there's no cleanup needed anymore? Should it do `pointer = nil` here?

Reply:
Fine to remove. The cleanup is done by `gifski_finish()`

Reply:
But what if it encounters an error and then `gifski_finish()` is not called.